packages/react/src/components/Chat/ChatItem.tsx

```diff
@@ -35,13 +35,13 @@ export interface ChatItemProps extends UIComponentProps, ChildrenComponentProps
   attached?: boolean | 'top' | 'bottom'
 
   /** Chat items can have a gutter. */
-  gutter?: ShorthandValue
+  gutter?: ShorthandValue<BoxProps>
 
   /** Indicates whether the content is positioned at the start or the end. */
   contentPosition?: 'start' | 'end'
 
   /** Chat items can have a message. */
-  message?: ShorthandValue
+  message?: ShorthandValue<ChatMessageProps>
```

Comment:
Actually this should be `BoxProps`, see line 109. We can easily do some miss match here.. :(

Reply:
Great catch 👍 

I am bit disappointed that we don't have strict checks there, but at least we will be able to improve them and catch such cases